lumigator/python/mzai/backend/backend/services/datasets.py

```diff
@@ -18,7 +18,7 @@
 from backend.settings import settings
 
 GT_FIELD: str = "ground_truth"
-REQUIRED_EXPERIMENT_FIELDS: set[str] = {"examples"}
+REQUIRED_EXPERIMENT_FIELDS: set[str] = {"predictions"}
```

Comment:
fyi @aittalam

Reply:
Is this for a simple dataset upload? I think examples (or we should more generally call them "samples" at some point) are the only strict requirement for _uploading_ a dataset, while I agree that both predictions + gt are needed to run the evaluation

Reply:
yup, I was under the impression that we're changing the baseline field name to predictions as well, but samples makes a lot more sense. 

What should we have for all the field names just to get it down on "paper"? The way I had in my mental model after correcting for samples: 

You're uploading any given dataset: 
`samples` <-- your input data (perhaps `input_samples` in that case?)

You have your own ground truth: 
`samples` `ground_truth`

You've performed ground truth inference: 
`samples` `ground_truth` `prediction` (the generated gt)

Does that make sense?

Reply:
ah I guess that's just for being able to test with a dataset with predictions + gt? Please ignore me in that case :-)

Reply:
Yep! I'd keep `samples` for simplicity - and then we should make sure things stay consistent across all tests / docs

Reply:
ok so baseline input is `samples` for `REQUIRED_EXPERIMENT_FIELDS` and then we extend as needed for jobs. makes sense to me!

Reply:
perfect, TY!